src/lager_transform.erl

```diff
@@ -267,8 +267,11 @@ handle_args(DefaultAttrs, Line, [Arg1, Arg2]) ->
 handle_args(DefaultAttrs, _Line, [Attrs, Format, Args]) ->
     {concat_lists(Attrs, DefaultAttrs), Format, Args}.
 
-make_varname(Prefix, Line) ->
-    list_to_atom(Prefix ++ atom_to_list(get(module)) ++ integer_to_list(Line)).
+make_varname(Prefix, Loc) ->
+    list_to_atom(Prefix ++ atom_to_list(get(module)) ++ integer_to_list(line_from_loc(Loc))).
```

Comment:
There's a way to improve this call (using `erl_anno`). I'll create a new pull request for this.